ocs_ci/templates/ocp-deployment/install-config-vsphere-upi.yaml.j2

```diff
@@ -3,6 +3,9 @@ baseDomain: {{ base_domain | default('qe.rh-ocs.com') }}
 metadata:
   name: '{{ cluster_name }}'
 networking:
+  clusterNetwork:
+    - cidr: 10.128.0.0/14
+      hostPrefix: 23
```

Comment:
do we plan more than 254 hosts?

Reply:
What about have this value configurable via config?

```
    - cidr:  {{ cluster_cidr | default("10.128.0.0/14") }}
      hostPrefix: {{ cluster_host_prefix | default(23) }}
```

this will give us capability to change this via config value in ENV_DATA section. WDYT? But not sure if we will need to change this value?

If it's needed for scale, we can have specific config for them maybe, which they will use when deploying, so maybe we can have just condition if cluster_cidr data are in config and if so add this section in networking.

Reply:
Done

Reply:
> do we plan more than 254 hosts?

In recent commit , values are configurable.